### ISSUE TITLE
Link between EQB and CDC

### ISSUE THREAD

**Issue by cessda-bitbucket-importer** [cessda/cessda.cdc.versions#137]
**Original report on BitBucket by John Shepherdson (GitHub: [john-shepherdson](https://github.com/john-shepherdson)).**

----------------------------------------

**SEE ALSO #339**

‌

Dear John,

In order to allow our End users to access further study-related information in CDC, we have integrated “Link to CESSDA Data Catalogue” in our EQB model and have implemented it in our User Interface which dynamically links the study \(that belongs to a question in EQB\) to the corresponding CDC page.

Currently, we use a workaround creating links such as [https://datacatalogue.cessda.eu/?q=%2210.4232/1.11753%22](https://datacatalogue.cessda.eu/?q=%2210.4232/1.11753%22), but this leads to the result with one result, not the detailed study page \([https://datacatalogue.cessda.eu/detail?q=%22GESIS\_\_oai:dbk.gesis.org:DBK/ZA4614%22](https://datacatalogue.cessda.eu/detail?q=%22GESIS__oai:dbk.gesis.org:DBK/ZA4614%22)\), and is not failsafe. In order to implement it, we need the support from CDC.

This could be implemented as follows:

In EQB, we create a link with a prefix such as “[https://datacatalogue.cessda.eu/](https://datacatalogue.cessda.eu/)dataset/doi/” and append the doi that we find in the EQB study metadata.

In CDC, you create a router in the react/searchkit application, that leads directly to the detail view of the study corresponding to the doi.

In other words: we need a way to directly access the CDC detail page of a study while we currently only have the DOI of that study.

Do you think you could provide such functionality?

If you need further information or if you think a meeting would be necessary to clarify questions, please let me know.

Or you can contact our developer \(Thomas Krämer \(CC\)\) via E-Mail or Slack, if you have any questions.

Thank you.

All the best,

Esra
